Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Sliders/PinchSlider.cs

```diff
@@ -33,20 +35,66 @@ public GameObject ThumbRoot
             }
         }
 
-        [Range(0, 1)]
+        #region MyValue 
+        [SerializeField]
+        private int minValue = -4;
```

Comment:
If wholeNumbers is false, should we maybe support having non-whole numbered min/max values?

Oh actually! Maybe thinking more in general, the wholeNumbers thing seems to add a little more complexity around the usage of the data output from this component - I'm asking this just to gauge impact, but if we didn't have the wholeNumbers bit at all here (and just had it always return a float value between min and max value), would this be a much worse developer experience? (i.e. on the caller side, is it too much Math.round/Math.floor-ing?